src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/remove_convert.cpp

```diff
@@ -101,6 +98,13 @@ class RemoveUselessConvertCPUTest : public testing::WithParamInterface<RemoveCon
 };
 
 TEST_P(RemoveUselessBF16ConvertCPUTest, CompareWithRefs) {
+    ElementType inType;
+    InputShape inputShape;
+    std::tie(inType, inputShape) = this->GetParam();
+    if (inType == ElementType::bf16) {
```

Comment:
Minor: We can move this config to bf16 test instantiation instead of hardcode :)

Reply:
@iefode  thanks your suggestion, I will fix it

Reply:
fixed it